packages/aragon-wrapper/src/index.js

```diff
@@ -218,7 +234,7 @@ export default class Aragon {
           apps.map(async (app) => Object.assign(
             app,
             await this.apm.getLatestVersionForContract(app.appId, app.codeAddress)
-              .catch(() => ({}))
+              .catch(() => getAragonOSAppInfo(app.appId)) // for unpublished apps we check local mapping
```

Comment:
@bpierre this should give us the ACL and its artifact as an app in the `apps` observable, so we can remove the some of the hardcoded roles in aragon/aragon :)

Reply:
Yep, I was also thinking whether it would be beneficial to add the kernel to the app list as well.

Reply:
Was thinking about that too... not sure if it really belongs in the same list as it's not really an "app", but that would likely be the most convenient.

Another alternative is a `kernel` observable that returns the artifact and address of the kernel.